rotkehlchen/db/ethtx.py

```diff
@@ -156,10 +156,10 @@ def get_ethereum_transactions(
         cursor = self.db.conn.cursor()
         query, bindings = filter_.prepare()
         if has_premium:
-            query = 'SELECT * FROM ethereum_transactions ' + query
+            query = 'SELECT DISTINCT ethereum_transactions.tx_hash, timestamp, block_number, from_address, to_address, value, gas, gas_price, gas_used, input_data, nonce FROM ethereum_transactions ' + query  # noqa: E501
```

Comment:
why are you making this change here and below?

Why `DISTINCT` and why spell out all fields?

Reply:
Hmm okay would have to see how this plays in practise. Thanks